src/main/java/com/mattmalec/pterodactyl4j/client/managers/DatabaseAction.java

```diff
@@ -0,0 +1,13 @@
+package com.mattmalec.pterodactyl4j.client.managers;
+
+import com.mattmalec.pterodactyl4j.PteroAction;
+import com.mattmalec.pterodactyl4j.client.entities.Database;
+
+import java.util.List;
+
+public interface DatabaseAction extends PteroAction<Database> {
```

Comment:
This should also be following client conventions and should extend the generic DatabaseAction since both the application and client require name and allowed networks. 

```suggestion
public interface ClientDatabaseAction extends PteroAction<ClientDatabase> implements DatabaseAction {
```